hi.yml

```diff
@@ -0,0 +1,392 @@
+      compact: कॉम्पैक्ट मोड
+      profanity: अश्लीलता फ़िल्टर
+    footer: अपने कनेक्शन को प्रबंधित करने के लिए नीचे दिए गए बटन और ड्रॉपडाउन का उपयोग करें।
+  selects:
+    placeholder: 🛠️ नीचे से अपने कनेक्शन को प्रबंधित करने के लिए एक विकल्प का चयन करें।
+
+hub:
+  notFound: <:no:977600033740431410> हब नहीं मिल सका। कृपया सुनिश्चित करें कि आपने सही हब नाम दर्ज किया है।
+  notFound_mod: <:no:977600033740431410> हब नहीं मिल सका। कृपया सुनिश्चित करें कि आपने सही हब नाम दर्ज किया है और आप हब के मालिक या मॉडरेटर हैं।
+  alreadyJoined: <:no:977600033740431410> आपने पहले ही **{{hub}}** से {{{channel}}} से जुड़ लिया है!
+  invalidChannel: <:no:977600033740431410> इन्वैलिड चैनल। केवल पाठ और थ्रेड चैनल्स का समर्थन किया जाता है!
+  invalidImgurUrl: <:no:977600033740431410> आइकन या बैनर का इन्वैलिड URL दिया गया है। कृपया सुनिश्चित करें कि आपने एक वैलिड Imgur छवि URL दर्ज किया है जो एक गैलरी या एल्बम नहीं है।
+  join:
+    success: सफलतापूर्वक **{{hub}}** से {{{channel}}} से जुड़ लिया गया है! अपने कनेक्शन को प्रबंधित करने के लिए `/connection` का उपयोग करें। और हब छोड़ने के लिए `/hub leave` का उपयोग करें।
+
+  servers:
+    total: "वर्तमान जुड़े हुए सर्वर: {{from}}-{{to}} / **{{total}}**"
+    noConnections: <:no:977600033740431410> अब तक कोई सर्वर इस हब से जुड़ा नहीं है। `/hub join` का उपयोग करके इस हब से जुड़ें।
+    notConnected: <:no:977600033740431410> वह सर्वर **{{hub}}** का हिस्सा नहीं है।
+    connectionInfo: |
+      चैनल: #{{{channelName}}} `({{channelId}})`
+      जुड़ा गया: {{{joinedAt}}}
+      इनवाइट: {{{invite}}}
+      कनेक्टेड: {{connected}}
+
+  create:
+    modal:
+      title: हब बनाएं
+      name:
+        label: हब का नाम
+        placeholder: अपने हब के लिए एक नाम दर्ज करें।
+      description:
+        label: विवरण
+        placeholder: अपने हब के लिए विवरण दर्ज करें।
+      icon:
+        label: आइकन URL
+        placeholder: एक Imgur छवि URL दर्ज करें।
+      banner:
+        label: बैनर URL
+        placeholder: एक Imgur छवि URL दर्ज करें।
+
+    maxHubs: <:no:977600033740431410> You have reached the maximum number of hubs (3) you can create. Please delete a hub before creating another one.
+    invalidName: <:no:977600033740431410> Invalid hub name. It must not contain `discord`, `clyde` or \`\`\` . Please choose another name.
+    nameTaken: <:no:977600033740431410> This hub name is already taken. Please choose another name.
+    success: |
+      ### Hub Created!
+
+      Congratulations! Your private hub, **{{name}}**, has been successfully created.
+      To join, create an invite using `/hub invite create` and share the generated code. Then join using `/hub join`.
+
+      - **Generate invite:** `/hub invite create`
+      - **Go public:** `/hub manage`
+      - **Join hub:** `/hub join`
+      - **Edit hub:** `/hub manage`
+      - **Add moderators:** `/hub moderator add`
+
+      Learn more about hubs in our [guide](https://discord-interchat.github.io/docs). Join the [support server]({{{support_invite}}}) for help.
```

Comment:
Great work, @notanerdstudent! While reviewing, I noticed a small detail that was missed. 😅 
![image](https://github.com/Discord-InterChat/locales/assets/73829355/acaf8257-cb7a-415d-932e-64726420c478)

Reply:
no u dont